calico_containers/tests/st/test_arg_parsing.py

```diff
@@ -2,15 +2,15 @@
 
 from test_base import TestBase
 from docker_host import DockerHost
-
+from nose.plugins.attrib import attr
 
 class TestArgParsing(TestBase):
+    @attr('slow')
```

Comment:
Very surprising that this test is slow!  All it does is run calicoctl a bunch of times...